pandas/tests/reshape/test_reshape.py

```diff
@@ -218,36 +218,46 @@ def test_multiindex(self):
 
 class TestGetDummies(object):
 
-    sparse = False
+    @pytest.fixture
+    def df(self, dtype):
+        return DataFrame({'A': ['a', 'b', 'a'],
+                          'B': ['b', 'b', 'c'],
+                          'C': [1, 2, 3]}, dtype=dtype)
 
-    def setup_method(self, method):
-        self.df = DataFrame({'A': ['a', 'b', 'a'],
-                             'B': ['b', 'b', 'c'],
-                             'C': [1, 2, 3]})
+    @pytest.fixture(params=['uint8', 'float64'])
```

Comment:
cycle thru more dtypes here that are valid (doesn't have to be all, but include int64, bool, IOW valid for both sparse/dense)

Reply:
add None as well

Reply:
we should prob raise on ``object`` dtype I think.